wherehows-web/app/routes/application.js

```diff
@@ -77,7 +77,10 @@ export default Route.extend(ApplicationRouteMixin, {
    */
   sessionAuthenticated() {
     this._super(...arguments);
-    this._loadCurrentUser();
+    this._loadCurrentUser().then(() => {
```

Comment:
AFAIK, you shouldn't need to do this. Ember Simple Auth automatically retries a transition that was attempted before the user was authenticated, after authentication. Might be better to see why that no longer works in prod. It actually seems to work locally when I did a 2 sec test. The application route mixin should have it's `sessionAuthenticated ` method invoked, and that should have a reference to the previous transition in an `attemptedTransition` property.  Also, curious what happens if the `previousRouteName` was `/logout`?